src/polkitbackend/polkitbackendauthority.c

```diff
@@ -1591,3 +1598,24 @@ polkit_backend_authority_log (PolkitBackendAuthority *authority,
 
   g_free (message);
 }
+
+void
+polkit_backend_authority_set_log_level (const gchar *level)
+  {
+    if (g_strcmp0 (level, "debug") == 0)
+      polkit_authority_log_level = (guint) LOG_LEVEL_DEBUG;
+    else if (g_strcmp0 (level, "info") == 0)
+      polkit_authority_log_level = (guint) LOG_LEVEL_INFO;
+    else if (g_strcmp0 (level, "notice") == 0)
+      polkit_authority_log_level = (guint) LOG_LEVEL_NOTICE;
+    else if (g_strcmp0 (level, "warning") == 0)
+      polkit_authority_log_level = (guint) LOG_LEVEL_WARNING;
+    else if (g_strcmp0 (level, "err") == 0)
```

Comment:
Some options are abbreviated, some are not. Inconsistency brings confusion.

Reply:
Yeah these mixed abbreviations are very strange and I don't understand why they are named like that, however, that's just how syslog levels are called and they are pretty much the standard, and how they are called in the LogControl1 protocol, in all log parsers and viewers like journald and journalctl (ie: you do journalctl -p err rather than -p error) - I think consistency between ways to configure and use this and across the ecosystem is important here, more than aesthetic on an individual project level, although if I could go back in time I'd fix them to be consistently named:

https://man7.org/linux/man-pages/man3/syslog.3.html
https://www.freedesktop.org/software/systemd/man/latest/systemd.html#%24SYSTEMD_LOG_LEVEL
https://www.freedesktop.org/software/systemd/man/latest/org.freedesktop.LogControl1.html

Reply:
But the syslog manpage, from which LogControl got apparently inspired, communicates on the programmer level, not on the level of UI. Deriving textual parameters used directly by user from program's enum seems like a UI mistake made long ago. Do we want to continue in this evil? Who do we make this for? For admins who would be confused by numbers but not by half-words derived from enums?
I say we should use whole words for UI, program internals regardless, while we're at it.

Reply:
It needs to be the same yes, because again that's what user interfaces like systemctl and journalctl expose, so it's worse for usability to diverge: https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#service-log-level%20SERVICE%20%5BLEVEL%5D

What we can do, is accept both versions as input, as aliases. That's easy to do, I'll push shortly.